packages/medusa/src/repositories/price-list.ts

```diff
@@ -8,16 +8,17 @@ import {
 } from "typeorm"
```

Comment:
question: Why are we removing the typing from `relations`?

Reply:
Because we don’t pass a customerGroup to populate the IN we always use a string array. Passing an entity would mean that the method need to handle the transformation or that the consumer add a transformer to the findOperator